services/getstream/GetstreamService.js

```diff
@@ -11,7 +11,7 @@ class GetstreamService {
     const query = {
       limit,
       reactions: {own: true, recent: true, counts: true},
-      ranking: GETSTREAM_RANKING_METHOD,
+      // ranking: GETSTREAM_RANKING_METHOD,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `ranking` parameter from the query object may affect the order in which the feed items are returned. If the chronological order is desired, it's important to ensure that the Getstream service defaults to this ordering method when no ranking method is specified. Otherwise, you might need to explicitly set a chronological ranking method.
```diff
- // ranking: GETSTREAM_RANKING_METHOD,
+ ranking: 'chronological',
```
If the default behavior of Getstream service is indeed chronological ordering, then this change is fine and no action is needed.


<!-- This is an auto-generated comment by OSS CodeRabbit -->